__tests__/app.test.js

```diff
@@ -40,3 +41,35 @@ describe("/api/categories", () => {
 			});
 	});
 });
+describe("/api/reviews", () => {
+	test("GET 200 - should return with all the reviews", () => {
+		return request(app)
+			.get("/api/reviews/")
+			.expect(200)
+			.then((result) => {
+				expect(result.body.length > 0);
+				result.body.reviews.forEach((review) => {
+					expect(review).toMatchObject({
+						owner: expect.any(String),
+						title: expect.any(String),
+						review_id: expect.any(Number),
+						category: expect.any(String),
+						review_img_url: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						designer: expect.any(String),
```

Comment:
This assertion does not check if comment_count exists in each review object, which it definitely should.